### DISCUSSION TITLE
How to log values on blockchain

### DISCUSSION THREAD

**Discussion by btc8436954** [cosmos/community#32] [Chain Builders (Cosmos SDK)]
I run a new blockchain generated from ignite cli and cosmos sdk.
My running this command, I start the chain on my local:
`ignite chain serve`

How can I see the logs that I made on the blockchain to debug?

**Comment by facundomedica** [comment]
I believe it's better to ask in https://github.com/ignite/cli

**Reply by btc8436954** [reply]
Ok thanks
